src/llmcompressor/pipelines/basic/pipeline.py

```diff
@@ -42,7 +44,12 @@ def __call__(
 
         LifecycleCallbacks.calibration_epoch_start()
 
-        with calibration_forward_context(model):
+        with contextlib.ExitStack() as stack:
+            stack.enter_context(calibration_forward_context(model))
+
+            if dataset_args is not None and dataset_args.calibrate_moe_context:
```

Comment:
Maybe we can just use the calibration options moe_calibrate_all_experts and moe_calibrate_gated_acts directly, and always enter this context?

Reply:
Yeah we could. I was hoping to add these option in a follow-up